src/instrument/plans/lup_plan.py

```diff
@@ -9,15 +9,15 @@
 
 import logging
 
-logger = logging.getLogger(__name__)
-
-logger.info(__file__)
-
 from apstools.plans import lineup
 from bluesky import plan_stubs as bps
 from bluesky import plans as bp
 
-from ..framework.initialize import bec
+from ..initialize_bs_tools import bec
+
+logger = logging.getLogger(__name__)
+
+logger.info(__file__)
 
 
 def lup(detectors, motor, start, finish, npts=5, key="cen"):
```

Comment:
Can this be refactored to use `apstools.plans.lineup2`?

Reply:
Let's make this a separate pr